### PR TITLE
Added proper dockerfile

### PR DESCRIPTION
**Pull request by serge1peshcoff**
Fixes MEMB-613, WIP.
Superseded https://github.com/AEGEE/oms-frontend/pull/160

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -2,9 +2,10 @@ version: '3.2'
 
 services:
     oms-frontend:
```
Comment (linuxbandit) [inline docker/docker-compose.yml]: you should remove this now, it's already built in the dockerfile

Comment (serge1peshcoff) [reply docker/docker-compose.yml]: @linuxbandit remove what?

Comment (linuxbandit) [reply docker/docker-compose.yml]: the block oms-frontend, we can only keep oms-frontend-nginx

Comment (serge1peshcoff) [reply docker/docker-compose.yml]: ah yeah, I planned to leave only oms-frontend and remove oms-frontend-nginx totally, but forgot to do it. Will update tonight.

**Pull request merged**